region/norfair/east.json

```diff
@@ -3097,7 +3100,16 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "Morph" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "Morph",
+                {"cost": [
+                  {"heatFrames": 150}
```

Comment:
This must be a typo? Was it meant to be 1500?

Reply:
650 so good find.